webform_prefill.js

```diff
@@ -7,17 +7,13 @@
 
 Drupal.behaviors.webform_prefill = {
   attach: function (context, settings) {
-
-    if (!$.isFunction($.fn.formPrefill)) {
-      return;
-    }
-
-    var settings = Drupal.settings.webform_prefill || {
+    var settings = settings.webform_prefill || {
```

Comment:
Why can you be sure that `$.fn.formPrefill` exists but not `settings.webform_prefill`? Both of them should have been added in `webform_prefill_page_build()`?

Reply:
Hm, yes perhaps we can remove that.